modules/getting-started/ref-rhdh-sizing.adoc

```diff
@@ -1,28 +1,33 @@
 [id='ref-rhdh-sizing_{context}']
 = Sizing requirements for {product}
 
-Scalability of {product} requires significant resource allocation. The following table lists the sizing requirements for installing and running {product}, including both the {product-short} application and {product-short} database components.
+Scalability of {product} requires significant resource allocation. The following table lists the sizing requirements for installing and running {product}, including {product-short} application, database components, and Operator.
 
 .Recommended sizing for running {product}
-[cols="40%,30%,30%", frame="all", options="header"]
+[cols="25%,25%,25%,25%", frame="all", options="header"]
 |===
 |Components
 |{product} application
 |{product} database
+|{product} Operator
 
 |Central Processing Unit (CPU)
 |4 vCPU
 |2 vCPU
+|1 vCPU
 
 |Memory
 |16 GB
 |8 GB
+|256 Mi
 
 |Storage size
 |2 GB
 |20 GB
+|NA
 
 |Replicas
 |2 or more
 |3 or more
+|NA
```

Comment:
```suggestion
|1
```

I would say exactly 1 replica, as the operator is cluster-wide atm, and we want to have a single instance of it running.